{{cookiecutter.github_repository}}/tests/integration/test_auth.py

```diff
@@ -40,3 +41,77 @@ def test_user_login(client):
         'id', 'email', 'name', 'auth_token'
     ]
     assert set(expected_keys).issubset(response.data.keys())
+
+
+def test_user_password_change(client):
+    url = reverse('auth-password-change')
+    current_password = 'password1'
+    new_password = 'paSswOrd2.#$'
+    user = f.create_user(email='test@example.com', password=current_password)
+    change_password_payload = {
+        'current_password': current_password,
+        'new_password': new_password
+    }
+
+    client.login(user=user)
+    response = client.json.post(url, json.dumps(change_password_payload))
+    assert response.status_code == 204
+    client.logout()
+
+    url = reverse('auth-login')
+    credentials = {
+        'email': user.email,
+        'password': new_password
+    }
+    response = client.json.post(url, json.dumps(credentials))
+    assert response.status_code == 200
+    expected_keys = [
+        'id', 'email', 'name', 'auth_token'
+    ]
+    assert set(expected_keys).issubset(response.data.keys())
+
+    user.refresh_from_db()
+    assert user.check_password(new_password)
+
+
+def test_user_password_reset(client, mailoutbox):
+    url = reverse('auth-password-reset')
+    user = f.create_user(email='test@example.com')
+
+    response = client.json.post(url, json.dumps({'email': user.email}))
+    assert response.status_code == 204
+    assert len(mailoutbox) == 1
+
+
+def test_user_password_reset_and_confirm(client, settings, mocker):
```

Comment:
since this test checks password reset confirm, I have removed the individual test for the api.

Reply:
👍